src/main/kotlin/com/msd/robot/application/dtos/UpgradeDto.kt

```diff
@@ -0,0 +1,10 @@
+package com.msd.robot.application.dtos
+
+import com.msd.robot.domain.UpgradeType
+import java.util.*
+
+class UpgradeDto(
+    val transaction_id: UUID,
+    val `upgrade-type`: UpgradeType,
```

Comment:
Why are you using hyphens for the variable names? If it is for the JSON objects then I'm quite sure there annotations for that.